README.md

```diff
@@ -119,7 +119,7 @@ two environment variables:
 
 Documentation (built daily from the Github `master` branch) is available at:
 
-https://roman-cal-pipeline.readthedocs.io/en/latest/
+https://roman-pipeline.readthedocs.io/en/latest/
```

Comment:
Even though the code was corrected to clone from 'main' this still has 'master' here and in 2 other locations in this file.

Reply:
Good catch!